extpsdk/src/main/python/vantiqconnectorsdk.py

```diff
@@ -434,6 +434,9 @@ async def _perform_connection(self, do_pings: bool) -> string:
                     status = resp[_STATUS]
                     if status >= 300:
                         return _CONNECTION_FAILED
+                    # Sometimes we get a status message back.  If so & it's OK, wait for the config message
+                    raw_resp = await websocket.recv()
+                    resp = json.loads(raw_resp)
```

Comment:
This should probably be part of a loop. Otherwise, it looks like we don't handle a failure or second status message after the first status message.